src/views/evidence-upload.njk

```diff
@@ -85,6 +100,11 @@
       <p>
         <a href="{{ navigation.forward.href }}" class="govuk-link">Continue without adding documents</a>
       </p>
+      <form method="post">
```

Comment:
I suggest that work is moved to separate PR while this PR focuses only on linking evidence question page.